client/src/components/SearchPage/SearchField.js

```diff
@@ -0,0 +1,111 @@
+import React, {Component} from 'react';
+import './SearchField.css'
```

Comment:
Use css modules. Don't import CSS directly, it could lead to conflicts later.

In the latest version of react, all you need to for using CSS modules is to name any css file `<filename>.module.css`. After this you can import classes from the file in the usual css-modules way.